src/generic_core/user.ts

```diff
@@ -452,7 +452,7 @@ module Core {
           instancePromises.push(
               Core.RemoteInstance.create(this, instanceId, null)
                   .then((newInstance) => {
-                    this.instances_[instanceId] = newInstance;
+                    this.instances_[newInstance.instanceId] = newInstance;
```

Comment:
This shouldn't change behavior, right?  It seems like `newInstance.instanceId` should equal `instanceId`.